src/globalStyles/global.scss

```diff
@@ -552,7 +553,8 @@ h3.hr {
 }
 
 .inlineBlock {
-  display:inline-block;
+  float: right;
+  overflow: hidden;
```

Comment:
probably not a good idea to do this in a generic class like this

Reply:
I have implemented a new class .chartWrapper used in PlotsTab that uses the overflow:hidden style property.